poetry/console/commands/shell.py

```diff
@@ -0,0 +1,52 @@
+from os import environ
+from distutils.util import strtobool
+
+from .venv_command import VenvCommand
+
+
+class ShellCommand(VenvCommand):
+    """
+    Spawns a shell within the virtual environment.
+
+    shell [options]
+    """
+
+    help = """The <info>shell</> command spawns a shell, according to the
+<comment>$SHELL</> environment variable, within the virtual environment.
```

Comment:
Maybe we could add info about usage of "VIRTUAL_ENV"  herre:
```
The <info>shell</> command spawns a shell (according to the
+<comment>$SHELL</> environment variable) within the virtual environment (according to the
+<comment>$VIRTUAL_ENV</> environment variable) ....
```

Reply:
But the `VIRTUAL_ENV` variable shows the path to the currently activated virtual environment. It won't be read to activate, but it will be set afterwards.